root/components/common-macros.tt

```diff
@@ -5,17 +5,11 @@
 [%~ MACRO add_commas(n) GET n.chunk(-3).join(',') -%]
 [%~ MACRO replace(text, search, replace) BLOCK; text | replace(search, replace); END -%]
 [%~ MACRO url_escape(url) BLOCK; url | url; END -%]
-[%~ MACRO html_escape(url) BLOCK; url | html; END -%]
+[%~ MACRO html_escape(text) BLOCK; text | html; END ~%]
```

Comment:
Just a naming issue … The argument to `html_escape` can be a URL, but most often is some other text.